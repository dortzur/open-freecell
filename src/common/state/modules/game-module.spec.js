
import gameReducer, {startGame} from './game-module';

describe('legacy', () => {
  it('creates game 100', () => {

    const actionResult = startGame(100);
    expect(actionResult).toMatchSnapshot();
    expect(gameReducer(undefined, actionResult)).toMatchSnapshot();


  });
});
