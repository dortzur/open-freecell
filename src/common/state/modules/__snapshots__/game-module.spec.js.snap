// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`game-module creates game 100 1`] = `
Object {
  "payload": Object {
    "gameNumber": 100,
  },
  "type": "START_GAME",
}
`;

exports[`game-module creates game 100 2`] = `
Object {
  "cells": Array [
    Array [],
    Array [],
    Array [],
    Array [],
  ],
  "foundations": Array [
    Array [],
    Array [],
    Array [],
    Array [],
  ],
  "tableau": Array [
    Array [
      "AD",
      "QH",
      "QS",
      "6S",
      "3C",
      "2C",
      "8S",
    ],
    Array [
      "JS",
      "TS",
      "TH",
      "5H",
      "KS",
      "2H",
      "KC",
    ],
    Array [
      "4S",
      "5C",
      "7S",
      "6C",
      "JC",
      "KH",
      "9S",
    ],
    Array [
      "QC",
      "9D",
      "8H",
      "3D",
      "3S",
      "8C",
      "5S",
    ],
    Array [
      "7C",
      "4H",
      "9C",
      "4D",
      "TC",
      "7D",
    ],
    Array [
      "6D",
      "JD",
      "6H",
      "2S",
      "4C",
      "8D",
    ],
    Array [
      "JH",
      "KD",
      "AC",
      "9H",
      "5D",
      "QD",
    ],
    Array [
      "TD",
      "AH",
      "AS",
      "7H",
      "3H",
      "2D",
    ],
  ],
}
`;
